web/styles/rendered_markdown.css

```diff
@@ -458,6 +458,41 @@
         float: none;
     }
 
+    .message_inline_video {
+        position: relative;
+        margin-bottom: 5px;
+        height: 150px;
+        width: 250px;
+        display: inline-block;
+
+        &:hover {
+            &:not(.play_button_hidden)::after {
+                transform: scale(1.5);
+            }
+        }
+
+        &:not(.play_button_hidden)::after {
+            content: "";
+            background-image: url("../images/play_button.svg");
+            display: block;
+            width: 24px;
+            height: 24px;
+            position: absolute;
+            top: 63px;
+            left: 113px;
+            border-radius: 100%;
+            transition: transform 0.2s;
+        }
+
+        > video {
+            position: relative;
+            height: 150px;
```

Comment:
It might be good to note that these height & width attributes conform the player area to 16:9 (and maybe put them in width, height order), but that `object-fit: contain` maintains the aspect ratio of the original video without stretching or smooshing it.